### ISSUE TITLE
Iptc.Application2.Keywords problem

### ISSUE THREAD

**Issue by gigatavu** [LeoHsiao1/pyexiv2#18]
Iptc.Application2.Keywords returs only one keyword, although there are several keywords in the image.

**Comment by LeoHsiao1** [comment]
I haven't used this tag, and I'm trying to assign it:
```python
>>> img.modify_iptc({'Iptc.Application2.Keywords': ['hello', 'world']})
>>> img.read_iptc()['Iptc.Application2.Keywords']
'hello, world'
```

Do you want it to return a list of string, `['hello', 'world']` ?

**Comment by gigatavu** [comment]
The list would be better!

**Comment by gigatavu** [comment]
```
from pyexiv2 import Image
img = Image(r'/photo.jpg')
iptc = img.read_iptc()
print(iptc['Iptc.Application2.Keywords'])
img.close()

from iptcinfo3 import IPTCInfo
info = IPTCInfo('photo.jpg')
print(info['keywords'])
```

>>> python3 test.py
Summer
[b'River', b'Boat', b'Sweden', b'Summer']

**Comment by LeoHsiao1** [comment]
I will improve the code this weekend.

**Comment by LeoHsiao1** [comment]
I've improved the code so that it can read and write duplicate IPTC keys now.
I will release a new version in a few days.

Sample:
```python
>>> from pyexiv2 import Image
>>> img = Image(r'.\pyexiv2\tests\1.jpg')
>>> img.modify_iptc({'Iptc.Application2.Keywords': ['tag1', 'tag2', 'tag3']})
>>> img.read_iptc()['Iptc.Application2.Keywords']
['tag1', 'tag2', 'tag3']
>>> img.close()
```

**Comment by LeoHsiao1** [comment]
I have released a new version, [v2.2.0](https://github.com/LeoHsiao1/pyexiv2/releases/tag/v2.2.0), please try it.

**Comment by gigatavu** [comment]
Thanks! It works!!!
